packages/components/src/alignment-matrix-control/stories/index.story.tsx

```diff
@@ -24,10 +24,11 @@ const meta: Meta< typeof AlignmentMatrixControl > = {
 		'AlignmentMatrixControl.Icon': AlignmentMatrixControl.Icon,
 	},
 	argTypes: {
-		onChange: { action: 'onChange', control: { type: null } },
+		onChange: { control: { type: null } },
 		value: { control: { type: null } },
 	},
 	parameters: {
+		actions: { argTypesRegex: '^on.*' },
```

Comment:
This change introduces an error, but turns out [it was a Storybook problem](https://github.com/storybookjs/storybook/pull/24237), so I think it's safe to ignore it.

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'count')
```